### PR TITLE
Review for The Responsive Page First Milestone 

### PR DESCRIPTION
**Pull request by pountzas**
# Milestone 1

### On this milestone we had to create the following components for our website:

- navbar
- highlights section
- latest findings section
- footer

## Files to add
- index.html
- styles.css
- .github/workflows/linters.yml
- images/

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
# Status: Invalid

**Hello team 👋. I marked this project as invalid because of the following reasons:**

1). **Some linter checks are failing.**
2). **You have not provided a meaningful title and a brief summary for your pull request.**

**Fix the above and submit for another review.**

### Happy coding 🥂.

**You can reach me on slack (Rose Wanjohi).**

File: index.html [hunk 1]
```diff
@@ -4,12 +4,219 @@
...
+      <div class="yellow-bar flex">
+        <ul>
+          <li>Calling young digital talent in the Netherlands – Be recognized in the T500 →	</li>
+          
+        </ul>
+      </div>
+    </nav>
+
+    <main>
+      <div class="grid-one">
```
Comment (ioilmio) [inline index.html]: This section should be a `header` html semantic tag

File: index.html [hunk 2]
```diff
@@ -4,12 +4,219 @@
...
+        </div>
+      </div>
+      <div class="funding">
+        <div class="funding-title">
+          <a href="#"><h2>Latest funding rounds on Index.co →</h2> </a>
+          <div class="funding-tags">
+            <a href="#"><h6>See more</h6></a>
+            <a href="#"><h6>Get updates by email</h6></a>
+            <a href="#"><h6>Add info</h6></a>
+          </div>
```
Comment (ioilmio) [inline index.html]: This section should be a `nav` html semantic tag

File: index.html [hunk 3]
```diff
@@ -4,12 +4,219 @@
...
+        </div>
+        <div class="funding-list">
+
+          <div class="funding-element">
+            <img src="./images/image-funding.webp" alt="">
+            <div class="funding-h3">
+              <h3>Ringover</h3>
```
Comment (ioilmio) [inline index.html]: This and all the other titles in this section should also be a link tag

File: index.html [hunk 4]
```diff
@@ -4,12 +4,219 @@
...
+              <h4 class="money-text">$12M</h4>
+              <h5 class="money-text" >SERIES A</h5>
+            </div>
+            <div class="funding-links">
+              <a href="#" class="block">Telecommunications</a>
+              <a href="#" class="block">Expedition Growth Capital</a>
+            </div>
+            <div class="funding-desc">
+              <p>Ringover raises $12 million to bring enterprise communications to the cloud on VentureBeat
+              </p>
```
Comment (ioilmio) [inline index.html]: This and all the other text at the bottom of the article in this section should also be a link tag

File: index.html [hunk 5]
```diff
@@ -4,12 +4,219 @@
...
+            </div>
+          </div>
+        </div>
+      </div>
+    </main>
+
+    <footer class="footer">
+      <div class="first-footer">
```
Comment (ioilmio) [inline index.html:190]: this section should be a `nav` html semantic tag

File: index.html [hunk 6]
```diff
@@ -4,12 +4,219 @@
...
+          <li><a href="#">EVENTS</a></li>
+          <li><a href="#">ABOUT</a></li>
+          <li><a href="#">ADVERTISE</a></li>
+          <li><a href="#">JOBS</a></li>
+          <li><a href="#">CONTACT</a></li>
+        </ul>
+      </div>
+
+      <div class="second-footer">
+        <div class="contant-footer flex">
```
Comment (ioilmio) [inline index.html:211]: This one should be a `nav` html semantic tag

**Review by ioilmio — CHANGES_REQUESTED** [review]
# Hello @pountzas @Amalcxc 👋🏽 

### Your Project is almost done you need to implement some changes anyway:

# Required Changes ♻️ 

- ## Previous TSE ask you to improve your Pull Request title and description(important and mandatory)

- ### The header section should stick to the top when the user scrolls down
image

- ### You should replicate this red gradient overlay for your first article 
image

- ### Instead of using max-width for your media queries you could implement it using min-width on two screen size(e.g. 768px and 1024px) doing so you avoid repetition

-  ### Follow the comment I left for you in the review for more fixes to make

*For any doubts or clarification you can contact me on slack @illuminatosalvatore*

File: index.html (hunk 6 above)
Comment (Amalcxc) [reply index.html:211]: done

File: index.html (hunk 5 above)
Comment (Amalcxc) [reply index.html:190]: done

File: index.html (hunk 4 above)
Comment (Amalcxc) [reply index.html]: done

File: index.html (hunk 3 above)
Comment (Amalcxc) [reply index.html]: done

File: index.html (hunk 2 above)
Comment (Amalcxc) [reply index.html]: done

File: index.html (hunk 1 above)
Comment (Amalcxc) [reply index.html]: done

**Comment by pountzas** [comment]
The last commit has the linear gradient of the first highlight.

**Review by christianotieno — APPROVED** [review]
### Milestone Approved 🍾 
Hello @pountzas & @Amalcxc 👋 
 ##
This milestone has met all the requirements needed for approval. 👏 
Congratulations on the hard work done on this milestone. I liked the way you have incorporated responsiveness in the three main articles, the footer, and the navbar. Good job 👍 
I will go ahead and mark this as complete. Good luck with the next tasks 🥇 
##
### Happy coding 💻 
#### Christian Otieno (CR)

**Pull request merged**